lib/bolt/executor.rb

```diff
@@ -98,6 +98,7 @@ def batch_execute(targets)
     end
 
     def run_command(targets, command, options = {}, &callback)
+      @logger.notice(options['_description']) if options.key?('_description')
```

Comment:
I think this should just be worked into the next message and follow the verbosity of it.